doc/releases/changelog-0.35.0.md

```diff
@@ -308,19 +460,14 @@
   ([0.5, 1.0],
    [X(1) @ X(0),
     X(2) @ X(0)])
+  >>> op.coeffs
+  [0.5, 1.0]
+  >>> op.ops
+  [X(1) @ X(0), X(2) @ X(0)]
```

Comment:
I get deprecation warnings here:

```
qml.operation.enable_new_opmath()
op = qml.X(0) @ (0.5 * qml.X(1) + qml.X(2))
op.coeffs
```

```
[/Users/isaac/Documents/pennylane/pennylane/ops/op_math/prod.py:498](https://file+.vscode-resource.vscode-cdn.net/Users/isaac/Documents/pennylane/pennylane/ops/op_math/prod.py:498): PennyLaneDeprecationWarning: Prod.coeffs is deprecated and will be removed in future releases. You can access both (coeffs, ops) via op.terms(). Also consider op.operands.
```

Reply:
Oh! Nvm this is expected based on https://github.com/PennyLaneAI/pennylane/pull/5164. Why is this in the new features section then? Was it implemented as a new feature for other reasons (feature parity with Hamiltonian) but will be removed?

Reply:
```suggestion
```
Yes let's remove them. We are trying to steer users way from `coeffs` and `ops` as it's less efficient. But this isn't set in stone just yet.